indico/core/signals/rh.py

```diff
@@ -18,6 +18,14 @@
 *result*.
 ''')
 
+before_check_access = _signals.signal('before-check-access', '''
+Executed right before `_check_access` of an `RH` instance has been called.
+The *sender* is the RH class, the current instance is passed in *rh*.
+If the signal returns ``True`` or ``False``, the original `_check_access` method
+will not be executed.  If multiple subscribers to the signal return
+contradictory results, ``False`` wins and access is denied.
+''')
```

Comment:
```suggestion
In case access is denied, a generic error message will be displayed.
For custom errors, raise `Forbidden` yourself while handling the signal.
''')
```